src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -282,13 +285,9 @@ describe('signals/incident-management/components/FilterForm', () => {
 
     const addressField = container.querySelector('input[type="text"][name="address_text"]');
 
-    await act(async () => {
-      fireEvent.change(addressField, { target: { value: 'Weesperstraat 113/117' } });
-    });
+    await act(async () => { fireEvent.change(addressField, { target: { value: 'Weesperstraat 113/117' } }); });
 
-    await act(async () => {
-      fireEvent.blur(addressField);
-    });
+    await act(async () => { fireEvent.blur(addressField); });
```

Comment:
Same here.

Reply:
Yes, without the test will break, this seems to be some `testing-library` magic